3p/facebook.js

```diff
@@ -72,6 +93,7 @@ function getCommentsContainer(global, data) {
   container.setAttribute('data-numposts', data.numposts || 10);
   container.setAttribute('data-colorscheme', data.colorscheme || 'light');
   container.setAttribute('data-width', '100%');
+  container.setAttribute('data-height', '100%');
```

Comment:
Is this line change intentional? It looks like it affects `amp-facebook-comments` where this PR is for the `amp-facebook-page`